torch/distributed/checkpoint/default_planner.py

```diff
@@ -216,6 +216,47 @@ def transform_tensor(self, read_item: ReadItem, tensor: torch.Tensor):
         return narrow_tensor_by_index(tensor, read_item.dest_offsets, read_item.lengths)
 
 
+class _EmptyStateDictLoadPlanner(DefaultLoadPlanner):
```

Comment:
I had to move this out of format utils to avoid a circular dependency. I'm very open to suggestions on where a better home might be for this class